main.py

```diff
@@ -1,10 +1,55 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file_in:
+        data = json.load(file_in)
+    for player, info in data.items():
+        for key, value in info.items():
```

Comment:
it seems that this loop is extra
you can simply check `info["race"]` and `info["guild"]` then

Reply:
still not fixed